Extension/src/LanguageServer/protocolFilter.ts

```diff
@@ -74,7 +74,7 @@ export function createProtocolFilter(clients: ClientCollection): Middleware {
                 // didOpen), and first becomes visible.
             }
         },
-        didChange: (textDocumentChangeEvent, sendMessage) => {
+        didChange: async (textDocumentChangeEvent, sendMessage) => {
```

Comment:
Functions should only be marked `async` if they use `await` somewhere in the body.  Does this change the return type from `void` to `Promise<void>` without an `await`?

Reply:
Yes, this changes the return type from `void` to a `Promise<void>` automatically, and resolves the compilation error.  Alternatively, I could return a resolved `Promise<void>`, but this seems cleaner.